articles/virtual-desktop/TOC.yml

```diff
@@ -240,10 +240,16 @@
     href: /windows-server/remote/remote-desktop-services/virtual-machine-recs?context=/azure/virtual-desktop/context/context
   - name: Azure command-line interface (CLI)
     href: /cli/azure/
-  - name: Azure example scenarios
+  - name: Azure Virtual Desktop for the Enterprise
     href: /azure/architecture/example-scenario/wvd/windows-virtual-desktop
+  - name: FsLogix for the Enterprise
+    href: /architecture-center/docs/example-scenario/wvd/windows-virtual-desktop-fslogix-content.md
   - name: Email discovery
     href: /windows-server/remote/remote-desktop-services/rds-email-discovery
+  - name: Protect AVD with Azure Firewall
```

Comment:
```suggestion
  - name: Use Azure Firewall
```

Reply:
We're not supposed to use "AVD" in official documentation because it's not an approved acronym.

Reply:
can we use Azure Virtual Desktop with Azure Firewall?